src/pmdas/farm/help

```diff
@@ -0,0 +1,705 @@
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_17.cumulative_lifetime_unrecoverable_read_repeating Head 17
+Drive head 17: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_17.cumulative_lifetime_unrecoverable_read_unique Head 17
+Drive head 17: Cumulative lifetime  unrecoverable read unique for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_18.cumulative_lifetime_unrecoverable_read_repeating Head 18
+Drive head 18: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_18.cumulative_lifetime_unrecoverable_read_unique Head 18
+Drive head 18: Cumulative lifetime  unrecoverable read unique for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_19.cumulative_lifetime_unrecoverable_read_repeating Head 19
+Drive head 19: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_19.cumulative_lifetime_unrecoverable_read_unique Head 19
+Drive head 19: Cumulative lifetime  unrecoverable read unique for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_20.cumulative_lifetime_unrecoverable_read_repeating Head 20
+Drive head 20: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_20.cumulative_lifetime_unrecoverable_read_unique Head 20
+Drive head 20: Cumulative lifetime  unrecoverable read unique for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_21.cumulative_lifetime_unrecoverable_read_repeating Head 21
+Drive head 21: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_21.cumulative_lifetime_unrecoverable_read_unique Head 21
+Drive head 21: Cumulative lifetime  unrecoverable read unique for the drive
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_22.cumulative_lifetime_unrecoverable_read_repeating Head 22
+Drive head 22: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_22.cumulative_lifetime_unrecoverable_read_unique Head 22
+Drive head 22: Cumulative lifetime  unrecoverable read unique for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_23.cumulative_lifetime_unrecoverable_read_repeating Head 23
+Drive head 23: Cumulative lifetime unrecoverable read repeating for the drive
+
+@farm.ata.error_statistics.cum_lifetime_unrecoverable_errors_head_23.cumulative_lifetime_unrecoverable_read_unique Head 23
+Drive head 23: Cumulative lifetime  unrecoverable read unique for the drive
+
+@farm.ata.environment_statistics.current_temperature Current temperature
```

Comment:
'temperature' name repeats alot, consider a sub-namespace for temperature.* metrics here.